tests/benchmark/tests/release.py

```diff
@@ -1005,6 +1005,169 @@ def conda_libc_py_rosetta4_conda_release(kind, rosetta_dir, working_dir, platfor
+
+    extension = calculate_extension(platform, mode='release')
+
+    full_log += '\nRosetta binaries platform prefix: ' + platform_prefix + '\n'
+
+    payload_dir = working_dir + '/payload';  os.makedirs(payload_dir)
+    payload_bin_dir = payload_dir + '/bin';  os.makedirs(payload_bin_dir)
+    payload_database_dir = payload_dir + '/database';  os.makedirs(payload_database_dir)
+
+    TR(f'Copying build results...')
+    for part in 'external src'.split():
+        prefix = f'{rosetta_dir}/source/build/{part}/{platform_prefix}/'
+        for f in os.listdir(prefix):
+            if os.path.isfile(prefix + f):
+                shutil.copy(prefix + f, f'{payload_bin_dir}/{f}')
+                if f.endswith('.'+extension): os.symlink(f, f'{payload_bin_dir}/' + f[:-1-len(extension)])
+                elif f.endswith(extension): os.symlink(f, f'{payload_bin_dir}/' + f.partition('.default')[0])
+
+    TR(f'Copying Rosetta database...')
+    dir_util_module.copy_tree(f'{rosetta_dir}/database', payload_database_dir, update=False)
+
+    TR('Creating Rosetta package...')
+
+    results = {_StateKey_ : _S_passed_,  _ResultsKey_ : {},  _LogKey_ : full_log}
+    with open(working_dir+'/output.json', 'w') as f: json.dump(results, f, sort_keys=True, indent=2)  # makeing sure that results could be serialize in to json, but ommiting logs because they could take too much space
+
+    recipe_dir = working_dir + '/recipe';  os.makedirs(recipe_dir)
+
+    recipe = dict(
+        package = dict(
+            name    = 'rosetta',
+            version = version['version'],
+        ),
+        requirements = dict(
+            build = [f'python {platform["python"]}'],
+            host  = [f'python {platform["python"]}', 'zlib'],
+            run   = ['zlib'],
+        ),
+
+        about = dict(
+            home    = 'https://github.com/RosettaCommons/rosetta',
+            license = 'Rosetta license',
+            license_file = f'{rosetta_dir}/LICENSE.md',
+            summary = 'Rosetta, biomolecular modeling software package',
+            description = 'The Rosetta software suite includes algorithms for computational modeling and analysis of protein structures. It has enabled notable scientific advances in computational biology, including de novo protein design, enzyme design, ligand docking, and structure prediction of biological macromolecules and macromolecular complexes.',
+        ),
+    )
+
+    with open( recipe_dir + '/meta.yaml', 'w' ) as f: json.dump(recipe, f, sort_keys=True, indent=2)
+
+    with open( recipe_dir + '/build.sh', 'w' ) as f: f.write( _conda_rosetta_build_sh_template_.format(**locals()) )
+
+    release_kind = 'release' if config['branch'] == 'release' else 'devel'
+    conda_release_path = '{release_dir}/PyRosetta4/conda/{release_kind}'.format(release_dir=config['release_root'], release_kind = release_kind) # note that we intentionally using `PyRosetta4` path so all our releases is in the same channel
```

Comment:
This a path to our Conda channel. Historically it was first used to host PyRosetta packages and some supplement packages. I am not sure if it make sense to have more then one channel, but i am open to the idea if there is benefits, - thoughts?

Reply:
If it's purely server-side, there's probably no need to change it. 

I think it doesn't show up for users of conda. They add `https://conda.rosettacommons.org` or `https://conda.graylab.jhu.edu/` as their Conda channel and they get the pyrosetta or rosetta packages -- the "PyRosetta4" doesn't show up.

I don't think we need to have multiple Conda channels, so long as they appear to the users as "RosettaCommons" channels, rather than "PyRosetta" channels.